Automattic/wp-calypso
# SDK: Add simple CSS build process to our block builder

This PR adds a simple CSS build process to our already existing block builder script. It is an initial version that we can expand as we iterate. PR suggests the following:

* Using `node-sass` to build our block's `style.scss` file and output a CSS file in the `build` directory. Sourcemaps are on by default for now.
* Since the block builder now builds CSS, it now accepts a block directory instead of a block JS file.
* Renamed the existing Hello Dolly block main file to `index.js` to introduce consistency in block naming.
* Introduced a simple CSS file without much value.
* Used the new CSS selector in the existing block so it doesn't end up unused.

To test:
* Checkout this branch
* Run `node bin/create-scripts/block.js client/gutenberg/extensions/hello-dolly`
* Verify the JS is still built properly.
* Verify the CSS is also built properly in `/build/blocks-hello-dolly.css`.
